apps/web/components/CategoryItems/CategoryItems.tsx

```diff
@@ -20,14 +20,14 @@ export const CategoryItems = ({
   return (
     <>
       <GridContainer>
-        <Text variant="h3" as="h2" id={headingId}>
+        <Text variant="h2" as="h2" id={headingId}>
```

Comment:
This makes the title stick out way more than it used to be, is that something that's been decided upon? 
![image](https://github.com/island-is/island.is/assets/43557895/e29c2544-bccb-4ee7-9e9b-1247ca429959)

Reply:
these sections are now designed with header variant h2 for all sections